src/main/docker/keycloak.yml

```diff
@@ -13,11 +13,12 @@ services:
     networks:
       - kc-network
   keycloak:
-    image: bitnami/keycloak:14.0.0
+    image: quay.io/keycloak/keycloak:17.0.1
+    command: start-dev
     #volumes:
     #- ./realm-config:/opt/bitnami/keycloak/data/import
     environment:
-      KEYCLOAK_ADMIN_USER: admin
+      KEYCLOAK_ADMIN: admin
       KEYCLOAK_ADMIN_PASSWORD: admin
       KEYCLOAK_DATABASE_VENDOR: postgres
```

Comment:
Some of these environment variables not correct for the new image.
Please double check them.